bedrock/base/templates/macros.html

```diff
@@ -290,7 +290,13 @@ <h2 class="form-heading">
       <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
       <form id="send-to-device-form" action="{{ url('firefox.send-to-device-post') }}" method="post"{% if platform == 'select' %} class="dropdown"{% endif %}>
         <ul class="error-list hidden">
-          <li class="sms">{{ _('Sorry. This number isn’t valid. Please enter a U.S. phone number.') }}</li>
+          <li class="sms">
+          {% if l10n_has_tag('sendto_updates_bug1395342') %}
```

Comment:
Will an l10n tag work here, considering this is a shared macro?

Reply:
We're actually loading a file in the templates using sendto, so it should work (but worth testing)
https://github.com/mozilla/bedrock/search?utf8=%E2%9C%93&q=firefox%2Fsendto&type=

Reply:
Thanks @flodolo - I tested this on the current `/whatsnew` page and it does seem to be working 👍